### ISSUE TITLE
Change title writes non-translatable fields

### ISSUE THREAD

**Issue by georgobermayr** [getkirby/kirby#2775]
**Describe the bug**  
Changing the title of a page in the non-default language writes the content of non-translatable fields in the content file of the non-default language.

**To Reproduce**  
1. I have a field in the blueprint with `translate: false`
2. When I change the title of the page in a non-default language, the content of the non-translatable field gets written to the content file of the non-default language.
3. When I change the content of the non-translatable field in the default language, the content file in the non-default language is of course not updated and gets stuck with the older state and is from there on working with an older version of the content.

**Expected behavior**  
The content of the field should only be written in the default language

**Kirby Version**  
3.4.2

**Comment by afbora** [comment]
I can't reproduce this issue on 3.4.2 Starterkit with following bluerint 🤔 

Having the same issue with Kirby Starterkit? Also you can check the hooks and plugins.

````yaml
title: Simple Page
preset: page

fields:
  headline:
    type: text
    translate: false
  text:
    type: textarea
````

**Comment by georgobermayr** [comment]
Thank you, I did a bit more digging, and found, that the issue seems to be related with the writing of the field name in the blueprint:

```
testFieldCamel:
  type: text
  translate: false
testfieldlower:
  type: text
  translate: false
```

`testFieldCamel` gets written in the non-default language content file, `testfieldlower` not. So writing a field name in camel case seems to be the issue?

I think this is not the first time I encounter problems with camel cased field names in the blueprint. I would say that field names should always be just lowercase then?

**Comment by afbora** [comment]
Ahh this is really weird. I can reproduce the issue now with camelCase field name.

Also the good news is that @neildaniels has already solved this issue. It will be released in 3.5.0 🎉 https://github.com/getkirby/kirby/pull/2578 You can test with that PR.

**Comment by georgobermayr** [comment]
Thank you, good to know the issue will be resolved soon. I'm going for the safe solution for now and make the relevant field names lowercase 😉
